packages/account/src/configs.ts

```diff
@@ -1,4 +1,10 @@
-export const FUEL_NETWORK_URL =
+// #region FUEL_NETWORK_URL
```

Comment:
I've been personally trying to avoid snippet regions in `packages`, but not sure how to avoid this one. Are we only trying to avoid snippets in `packages` for tests? cc @FuelLabs/sdk-ts

Reply:
ad675e2de457f2e0d50d29b6e3a242a7d2cc325e

I have now removed the pre-defined constants section.
Clarity on your question @danielbate, would still be beneficial for me 😄

Reply:
I try and work as if our `packages` and `apps` could be in 2 entirely different repositories. So if we did want to separate them completely, we could. One of the reasons we have `docs-snippets` and not reference snippets in `fuel-gauge`. However this makes it tricky when we want to illustrate the shape of a type or object in the docs. Because then `apps` needs to reference something in `packages`. 🤔 

But yes you have resolved this :)